std/random.d

```diff
@@ -915,13 +915,7 @@ alias Mt19937_64 = MersenneTwisterEngine!(ulong, 64, 312, 156, 31,
     static assert(isUniformRNG!(Mt19937_64, ulong));
     static assert(isSeedable!Mt19937_64);
     static assert(isSeedable!(Mt19937_64, ulong));
-    // FIXME: this test demonstrates viably that Mt19937_64
-    // is seedable with an infinite range of `ulong` values
-    // but it's a poor example of how to actually seed the
-    // generator, since it can't cover the full range of
-    // possible seed values.  Ideally we need a 64-bit
-    // unpredictable seed to complement the 32-bit one!
-    static assert(isSeedable!(Mt19937_64, typeof(map!((a) => (cast(ulong) unpredictableSeed))(repeat(0)))));
```

Comment:
This is a great change, but it should be in a separate patch that follows the update to `unpredictableSeed` itself.